ios/FluentUI/Drawer/DrawerPresentationController.swift

```diff
@@ -126,7 +126,7 @@ class DrawerPresentationController: UIPresentationController {
         // For animated presentations presented view must be inside contentView to not slide over navigation bar/toolbar
         if presentingViewController.transitionCoordinator?.isAnimated == true {
             // Avoiding content animation due to showing of the keyboard (when presented view contains the first responder)
-            presentedViewController.view.frame = contentView.bounds
+            presentedViewController.view.frame = frameForPresentedViewController(in: contentView.bounds)
```

Comment:
can you just verify side drawers(left/right) when they rotate it doesn't show weird black box when animating?

Reply:
It seems the GIF does not show all the validation. But I did validate rotating from both sides while launching the drawer from both orientations (vertical/horizontal).